### DISCUSSION TITLE
Playlists by region

### DISCUSSION THREAD

**Discussion by fershady** [iptv-org/community#186] [Ideas]
I would like to get a playlist by region like :
https://iptv-org.github.io/iptv/region/<region_code>.m3u

And the output is sort by categories.

**Comment by what-a** [comment]
What exactly do you mean by "region?"

**Reply by fershady** [reply]
I refer to this : 
.readme/supported-region-codes.md
https://github.com/iptv-org/iptv/blob/7a648facb1d13662591f633289fe33106f19759c/.readme/supported-region-codes.md

scripts/data/regions.json
https://github.com/iptv-org/iptv/blob/7a648facb1d13662591f633289fe33106f19759c/scripts/data/regions.json

**Comment by BellezaEmporium** [comment]
Request fulfilled.
